portal-api/api-tests/v1/users/reset-password.api-test.js

```diff
@@ -28,28 +34,32 @@ describe('password reset', () => {
   beforeEach(async () => {
     wipeDB.deleteUser(MOCK_USER);
     await as(loggedInUser).post(MOCK_USER).to('/v1/users');
+
+    jest.clearAllMocks();
   });
 
-  afterEach(() => {
+  afterAll(() => {
     jest.clearAllMocks();
   });
 
-  it('should return success=false for non-existant email', async () => {
-    const reset = await resetPassword('no.user@email.com');
-    expect(reset).toMatchObject({ success: false });
+  it('should not send an email for non-existant email', async () => {
+    await resetPassword('no.user@email.com');
+    expect(sendEmail).toHaveBeenCalledTimes(0);
```

Comment:
```suggestion
    expect(sendEmail).not.toHaveBeenCalled();
```
Same functionality, reads ever so slightly better in my opinion